manifests/crds/analysis-run-crd.yaml

```diff
@@ -754,11 +754,619 @@ spec:
+                                                                type: string
+                                                            required:
+                                                            - name
+                                                            - value
+                                                            type: object
+                                                          type: array
+                                                        path:
+                                                          type: string
+                                                        port:
+                                                          anyOf:
+                                                          - type: string
+                                                          - type: integer
+                                                        scheme:
+                                                          type: string
+                                                      required:
+                                                      - port
+                                                      type: object
+                                                    initialDelaySeconds:
+                                                      format: int32
+                                                      type: integer
+                                                    periodSeconds:
+                                                      format: int32
+                                                      type: integer
+                                                    successThreshold:
+                                                      format: int32
+                                                      type: integer
+                                                    tcpSocket:
+                                                      properties:
+                                                        host:
+                                                          type: string
+                                                        port:
+                                                          anyOf:
+                                                          - type: string
+                                                          - type: integer
+                                                      required:
+                                                      - port
+                                                      type: object
+                                                    timeoutSeconds:
+                                                      format: int32
+                                                      type: integer
+                                                  type: object
+                                                resources:
+                                                  properties:
+                                                    limits:
```

Comment:
We need to remove the validation on the limits/requests of the ephemeral containers like we do for regular containers

Reply:
Nice catch. We also need to do this for initContainers too.